Code/Tools/Libs/ToolsFoundation/Reflection/Implementation/ReflectedTypeStorageAccessor.cpp

```diff
@@ -224,11 +224,11 @@ ezInt32 ezReflectedTypeStorageAccessor::GetCount(const char* szProperty) const
   if (m_pMapping->m_PathToStorageInfoTable.TryGetValue(szProperty, storageInfo))
   {
     if (storageInfo->m_Type == ezVariant::Type::Invalid)
-      return false;
+      return -1;
 
     const ezAbstractProperty* pProp = GetType()->FindPropertyByName(szProperty);
     if (pProp == nullptr)
-      return false;
+      return -1;
```

Comment:
it looks weird that the type of these two return values has changed, but nothing else in this function

Reply:
The function returns an int and -1 is the error case so it was simply wrong before.